src/static/pycontw-2020/assets/keynotes/wenyu-su.json

```diff
@@ -0,0 +1,29 @@
+{
+    "photo": "pycontw-2020/assets/keynotes/wenyu-su.png",
+    "speaker": {
+        "en": {
+            "name": "Wen Yu Su",
+            "bio": ""
```

Comment:
Gina just finished her final revision. Please update it when you have time.

Reply:
Added it. Thank you.